src/plugins/browser-console.ts

```diff
@@ -73,7 +75,13 @@ export class BrowserConsole {
 
   private enforceMessagesLimit() {
     if (this.messages.length > defaultMessageLimit) {
-      this.messages.splice(0, 1);
+      // drop logs type first if they exceed the limit
+      const lastLogIndex = this.messages.findIndex(m => m.type === 'log');
```

Comment:
I don't think we need to do this. With 1000 messages on the line which should be more than enough for successful journeys to be filtered on later, this is not necessary and an expensive operation for each console log statements.

Reply:
Done